src/renderer/features/settings/components/window/discord-settings.tsx

```diff
@@ -166,6 +166,61 @@ export const DiscordSettings = () => {
                 postProcess: 'sentenceCase',
             }),
         },
+        {
+            control: (
+                <Switch
+                    checked={settings.linkMusicbrainz}
+                    onChange={(e) => {
+                        setSettings({
+                            discord: {
+                                ...settings,
+                                linkMusicbrainz: e.currentTarget.checked,
+                            },
+                        });
+                    }}
+                />
+            ),
+            description: t('setting.discordLinkMusicbrainz_description', {
+                context: 'description',
+                discord: 'Discord',
+                musicbrainz: 'musicbrainz',
+                postProcess: 'sentenceCase',
+            }),
+            isHidden: !isElectron(),
+            title: t('setting.discordLinkMusicbrainz', {
+                discord: 'Discord',
+                musicbrainz: 'musicbrainz',
+                postProcess: 'sentenceCase',
+            }),
+        },
+        {
+            control: (
+                <Switch
+                    checked={settings.linkLastfm}
+                    onChange={(e) => {
+                        setSettings({
+                            discord: {
+                                ...settings,
+                                linkLastfm: e.currentTarget.checked,
+                            },
+                        });
+                    }}
+                />
+            ),
+            description: t('setting.discordLinkLastFM_description', {
+                context: 'description',
+                discord: 'Discord',
+                lastfm: 'last.fm',
+                musicbrainz: 'musicbrainz',
+                postProcess: 'sentenceCase',
+            }),
+            isHidden: !isElectron(),
+            title: t('setting.discordLinkLastFM', {
+                discord: 'Discord',
+                lastfm: 'last.fm',
+                postProcess: 'sentenceCase',
+            }),
+        },
```

Comment:
It feels a bit weird that this is multiple individual checkboxes (instead of something like a single select "discord/last.fm")